catalog/core/catalog-core-standardframework/src/main/resources/OSGI-INF/blueprint/fedstrategy.xml

```diff
@@ -29,7 +29,7 @@
 
     <bean id="cacheThreadPool" class="java.util.concurrent.Executors"
           factory-method="newFixedThreadPool">
-        <argument value="$[org.codice.ddf.system.threadPoolSize]"/>
+        <argument value="${org.codice.ddf.system.threadPoolSize}"/>
```

Comment:
❗️ Don't think this will work since this namespace the place holder https://github.com/codice/ddf/blob/master/catalog/core/catalog-core-standardframework/src/main/resources/OSGI-INF/blueprint/blueprint.xml#L25 .